src/game.cpp

```diff
@@ -12799,7 +12799,7 @@ void game::update_stair_monsters()
             critter.spawn( dest );
             while (tries < creature_push_attempts) {
                 tries++;
-                pushx = rng(-1, 1), pushy = rng(-1, 1);
+                pushx = rng(-1, 1); pushy = rng(-1, 1);
```

Comment:
Those two statements should go on separate lines - one statement per line, just like everywhere else.

Reply:
There are multiple variable declarations that go on the same row, like this:

https://github.com/CleverRaven/Cataclysm-DDA/blob/7c14418427e351016cc718f591dd0f909ed20883/src/activity_item_handling.cpp#L516

Should they be changed to separate lines too?

@kevingranade, mind if I ping you too.